RetailCoder.VBE/UI/RubberduckUI.fr.resx

```diff
@@ -429,13 +429,13 @@
     <value>Analyse des componsantes du projet...</value>
   </data>
   <data name="PromptLoadDefaultConfig" xml:space="preserve">
-    <value>{0}
+    <value>z {0}
 {1}
 
 {2}
 
 Restaurer les configurations par défaut?
-Attention: les valeurs personnalisées seront perdues.</value>
+Attention: les valeurs personnalisées seront perdues.    Your old file will be saved in '{3}'</value>
```

Comment:
@Hosch250 can you change that for 

    Le fichier original sera enregistré sous '{3}'.

and sync the PR? Thanks!